addons/website_event_booth/views/event_booth_templates.xml

```diff
@@ -0,0 +1,204 @@
+                <input type="hidden" name="event_booth_ids" t-att-value="event_booth_ids"/>
+                <div id="o_wevent_contact_details">
+                    <h4 class="mt32">
+                        <strong>Contact Details</strong>
+                    </h4>
+                    <div class="row form-group">
+                        <label class="col-form-label col-sm-auto">Name</label>
+                        <div class="col-sm">
+                            <input type="text" class="form-control" name="contact_name" required="True"
+                                   t-att-value="default_contact.get('name', '')"/>
+                        </div>
+                    </div>
+                    <div class="row form-group">
+                        <label class="col-form-label col-sm-auto">Email</label>
+                        <div class="col-sm">
+                            <input type="email" class="form-control" name="contact_email" required="True"
+                                   t-att-value="default_contact.get('email', '')"/>
+                        </div>
+                    </div>
+                    <div class="row form-group">
+                        <label class="col-form-label col-sm-auto">Phone <small>(Optional)</small></label>
+                        <div class="col-sm">
+                            <input type="tel" class="form-control" name="contact_phone"
+                                   t-att-value="default_contact.get('phone', '')"/>
+                        </div>
+                    </div>
+                    <div class="row form-group">
+                        <label class="col-form-label col-sm-auto">Mobile <small>(Optional)</small></label>
+                        <div class="col-sm">
+                            <input type="tel" class="form-control" name="contact_mobile"
+                                   t-att-value="default_contact.get('mobile', '')"/>
+                        </div>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <div class="col-sm-6 offset-sm-3 mt-5">
+                        <button type="submit" class="btn btn-primary btn-block font-weight-bold">Book My Booth</button>
+                    </div>
+                </div>
+            </form>
+        </t>
+    </template>
+
+    <template id="event_booth_registration_complete" name="Event Booth Registration Complete">
+        <t t-call="website_event_booth.event_booth_layout">
+            <div class="col-12 p-5">
+                <div class="row mb-3">
+                    <div class="col-12">
+                        <h3>Booth Registration Completed</h3>
+                        <span class="h4 text-muted" t-esc="event.name"/>
+                    </div>
+                </div>
+                <div class="d-flex flex-column">
+                    <span class="font-weight-bold">
+                        <t t-if="partner.name" t-esc="partner.name"/>
```

Comment:
Shouldn't we use real stored contact fields (contact_name, contact_email, ...) as they may be different from partner information ?